_posts/2023-03-06-NGINX-Dashboards-with-Fluent-Bit.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Simple NGINX dashboards with Fluent Bit and OpenSearch"
+authors:
+  - jdbright
+  - lioperry
+  - anurag_gup
+  - armarble
+date:   2023-03-06
+categories:
+  - technical-post
+meta_keywords: opensearch simple schema, observability schema, opensearch opentelemetry, OpenSearch 2.6
+meta_description: While users can currently send their logs to OpenSearch, there are no set formats or schemas, which can make sharing dashboards and alerts cumbersome. In this blog post, we talk about using Fluent Bit, a new simple schema, and OpenSearch with NGINX as the first example.
+---
+
+Fluent Bit is a graduated sub-project under the Cloud Native Computing Foundation (CNCF) Fluentd project umbrella. Fluent Bit integrates with hundreds of common tools such as Kafka, Syslog, Loki, and, of course, OpenSearch.
+
+While users can currently send their logs to OpenSearch, there are no set formats or schemas for the logs, which can make sharing dashboards and alerts cumbersome. In this blog post, we talk about using Fluent Bit, a new simple schema, and OpenSearch with NGINX as the example.
+
+## Simple Schema for Observability
+
+OpenSearch 2.6 introduced a standardization for conforming to a common and unified observability schema: Simple Schema for Observability. 
+
+Observability is a collection of plugins and applications that let you visualize data-driven events by using PPL/SQL/DQL to explore and query data stored in OpenSearch. With the schema in place, Observability tools can ingest, automatically extract, and aggregate data and create custom dashboards, making it easier to understand the system at a higher level.
+
+Simple Schema for Observability is inspired by [OpenTelemetry](https://opentelemetry.io/docs/) and Elastic Container Service ([ECS](https://github.com/elastic/ecs)) logs categorization and field sets and OpenTelemetry metadata.
```

Comment:
If we're referring to the AWS service, it has to be "Amazon Elastic Container Service (Amazon ECS)" on first mention. This sentence is still reading as confusing. Please revise for simplicity/clarity.